pilot/pkg/credentials/kube/secrets.go

```diff
@@ -156,19 +146,8 @@ func (s *CredentialsController) insertCache(user string, response error) {
 	}
 }
 
-// DisableAuthorizationForTest makes the authorization check always pass. Should be used only for tests.
-func DisableAuthorizationForTest(fake *fake.Clientset) {
-	fake.Fake.PrependReactor("create", "subjectaccessreviews", func(action k8stesting.Action) (bool, runtime.Object, error) {
-		return true, &authorizationv1.SubjectAccessReview{
-			Status: authorizationv1.SubjectAccessReviewStatus{
-				Allowed: true,
-			},
-		}, nil
-	})
-}
-
 func (s *CredentialsController) Authorize(serviceAccount, namespace string) error {
-	user := toUser(serviceAccount, namespace)
+	user := sa.MakeUsername(namespace, serviceAccount)
```

Comment:
I am not sure we want to bring in a complete new dependency to replace 1 line of code?

Reply:
This is a good question, I am not sure when I am doing this.

But after, the package provides many utilities, istio could use for authz